rnd/autogpt_builder/src/components/CredentialsProvider.tsx

```diff
@@ -0,0 +1,85 @@
+import AutoGPTServerAPI, { CredentialsMetaResponse } from "@/lib/autogpt-server-api";
+import { useRouter } from "next/navigation";
+import { createContext, useCallback, useEffect, useMemo, useState } from "react";
+
+const CredentialsProvidersNames = ["github", "google", "notion"] as const;
```

Comment:
- `as const`?
- Constants should be UPPER_SNAKE_CASE

Reply:
`as const` creates a readonly tuple as far as ts is concerned which makes sense for this array.
I'll rename it.